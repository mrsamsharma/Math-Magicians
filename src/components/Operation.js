import React from 'react';

// eslint-disable-next-line react/prop-types
export default function Operation({ operation, eventHandle }) {
  return (
    <button type="button" className="orange" onClick={eventHandle}>{operation}</button>
  );
}
