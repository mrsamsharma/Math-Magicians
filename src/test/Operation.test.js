import React from 'react';
import Operation from '../components/Operation';
import renderer from 'react-test-renderer';

it('Operation Component renders correctly', () => {
    const tree = renderer
      .create(<Operation />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });