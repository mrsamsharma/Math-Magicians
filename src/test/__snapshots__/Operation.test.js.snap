// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Operation Component renders correctly 1`] = `
<button
  className="orange"
  data-testid="+"
  onClick={Object {}}
  type="button"
>
  +
</button>
`;
